src/dune_rules/dune_package.mli

```diff
@@ -46,14 +46,18 @@ module Entry : sig
   val to_dyn : t Dyn.builder
 end
 
+type file =
```

Comment:
Weird to call this a file when it can be a directory. `path` is less misleading. Or even change the type to:

```
files : (Section.t * [`Directory | `File] * Path.t) list
```